.idea/uiprotect.iml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<module type="PYTHON_MODULE" version="4">
```

Comment:
Please do the removal in another separate PR as its not related to the new feature being added